src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -124,14 +123,11 @@ export const LongButtonText: StoryObj<Args> = {
       </Button>
     ),
   },
-  parameters: {
-    // Is flaky on chromatic, disabling for now
-    chromatic: { disableSnapshot: true },
-  },
 };
 
 export const Interactive: StoryObj<Args> = {
   args: {
+    duration: undefined,
```

Comment:
Does the duration end up being `undefined` or the default of 200 here?

Reply:
oh great question -- looks like Tippy is still getting `200`
<img width="574" alt="Screen Shot 2022-04-27 at 3 48 31 PM" src="https://user-images.githubusercontent.com/15840841/165618359-14541b37-58a6-443c-88f7-8daf35d8090d.png">

Reply:
Makes sense. What's the intent of setting this to `undefined` here? Should we remove the line?

Reply:
oh -- I set `duration : 0` in `defaultArgs` earlier (line 24), so thought it needed to be explicitly reset here the same way `visible` is being reset